pangeo_forge_runner/bakery/flink.py

```diff
@@ -250,6 +272,8 @@ def get_pipeline_options(
             save_main_session=True,
             # this might solve serialization issues; cf. https://beam.apache.org/blog/beam-2.36.0/
             pickle_library="cloudpickle",
+            parallelism=self.parallelism,
```

Comment:
I think these needed to be added to the dict only if they're not `None` so we should have an explicit check here.

Reply:
Good call I'll do that tomorrow

Reply:
Fixed in https://github.com/pangeo-forge/pangeo-forge-runner/pull/82/commits/611a590371affd94e3241af34ad131e412998747